crates/tabby-common/src/path.rs

```diff
@@ -89,4 +89,8 @@ impl ModelDir {
     pub fn ggml_q8_0_file(&self) -> String {
         self.path_string("ggml/q8_0.gguf")
     }
+
+    pub fn ggml_q8_0_v2_file(&self) -> String {
```

Comment:
updated llama.cpp requires re-converting all starcoder models, updating filepath to keep forward compatibility.